Gradata/tests/test_migration_002_event_identity.py

```
@@ -0,0 +1,184 @@
+        tags=["pre-migration"],
+    )
+    _null_identity_columns(brain)
+    _run_002(brain)
+
+    with _conn(brain) as conn:
+        row = conn.execute(
+            "SELECT event_id, device_id, content_hash, ts, type, source, data_json "
+            "FROM events WHERE type = 'TEST_HISTORICAL'"
+        ).fetchone()
+
+    event_id, device_id, content_hash, ts, ev_type, source, data_json = row
```

Ruff (0.15.10)

[warning] Unpacked variable `ts` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)